src/Popover.tsx

```diff
@@ -51,7 +51,7 @@ const PopoverContent = styled(BorderBox)`
   margin-right: auto;
   margin-left: auto;
   padding: ${get('space.4')};
-  background-color: ${get('colors.white')};
+  background-color: ${get('colors.bg.canvas')};
```

Comment:
Looks this should be `bg.overlay` ([reference](https://github.com/primer/css/blob/mkt/color-modes-whee/src/popover/popover.scss))
```suggestion
  background-color: ${get('colors.bg.overlay')};
```

Reply:
Also, usages of `get('popovers.colors.caret')` in this file should be replaces with `get('colors.border.overlay')`